src/components/ArticlePage.jsx

```diff
@@ -1,33 +1,58 @@
 import { useState, useEffect } from 'react'
 import { useParams } from 'react-router-dom';
-import { getArticles } from '../axios';
+import { getArticles, getComments } from '../axios';
+import CommentCard from './CommentCard'
 
 const ArticlePage = () => {
 
     const { article_id } = useParams();
-    const [currentArticle, setCurrentArticle] = useState({})
+    const [currentArticle, setCurrentArticle] = useState({});
     const [isLoading, setIsLoading] = useState(true);
 
+    const [commentList, setCommentList] = useState([]);
+    const [showComments, setshowComments] = useState(false);
+
+
     useEffect(() => {
         console.log("article-effect activated")
         getArticles(article_id)
             .then(response => {
                 setCurrentArticle(response.article)
                 setIsLoading(false)
             })
+        getComments(article_id)
+            .then(response => {
+                setCommentList(response.comments);
+            })
     }, [])
 
+    const handleOnShowCommentsClick = () => {
+        setshowComments(!showComments)
+    }
+
     const unformattedDate = new Date(currentArticle.created_at);
     const formattedDate = unformattedDate.toLocaleString('en-GB', {});
 
     if (isLoading) return <p>Loading articles</p>
     return (
         <article id="article-page">
             <h2>{currentArticle.title}</h2>
-                <img id='article-page-img' src={currentArticle.article_img_url} alt="{currentArticle.title}"/>
-            <p>Author: {currentArticle.author}</p>
-            <p>{formattedDate}</p>
+            <img id='article-page-img' src={currentArticle.article_img_url} alt="{currentArticle.title}" />
+            <p className='article-author'>Author: {currentArticle.author}</p>
+            <p className='article-time-stamp'>{formattedDate}</p>
             <p>{currentArticle.body}</p>
+            <h3>Comments</h3>
```

Comment:
A seemingly small thing, but I wanted to commend you for utilizing the Headings in the correct order. I know that your H1 is in the Header, and this is a really good thing to do because Screen Readers can understand the flow of the page alot better like this 👍